addons/im_livechat/static/src/embed/frontend/boot_service.js

```diff
@@ -24,6 +24,7 @@ export const livechatBootService = {
         const target = this.getTarget();
         const root = makeRoot(target);
         makeShadow(root).then((shadow) => {
+            env.services["discuss.rtc"].rootEl = shadow;
```

Comment:
Not the best but it works at least, as long as RTC service is used in one and only one context at a time.